azure-pipelines.yml

```diff
@@ -32,8 +32,8 @@ schedules:
     include:
     - main
   always: true
-- cron: "0 10-22 * * *"
-  displayName: Hourly build (6am - 6pm EST)
+- cron: "0 10-02 * * *"
```

Comment:
Is the cron setting supposed to be **02** here? If the desired end time is 11 PM EST, then you'd need to add 5 to get the UTC equivalent, which would be 4 AM (which in turn means 10 should be changed to 11 since 6 AM EST = 11 AM UTC).

The previous settings were valid for EDT which is only UTC+4 instead of UTC+5 (unless we don't want to bother adjusting for seasonal time changes)

Reply:
It looks like Azure pipelines does not automatically support seasonal time changes. We could potentially change it by hand once a year.

I can update so it accounts for the seasonal change.